CHANGELOG.md

```diff
@@ -27,7 +27,7 @@
 
 ### Development
 
-- ci: update github workflows to use latest version for depricated actions (checkout, setup-python, upload-artifact, download-artifact) #1967 @sjha2048
+- ci: update github workflows to use latest version of actions that were using a deprecated version of node #1967 #2003 @sjha2048 @Ana06
```

Comment:
should we add https://github.com/mandiant/capa-rules/pull/883 here as well?

Reply:
sure, if you want